src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -231,13 +231,29 @@ class JesBackend extends Backend with LazyLogging {
     }
   }
 
-  private def runWithJes(backendCall: BackendCall, command: String, jesInputs: Seq[JesInput], jesOutputs: Seq[JesOutput]): ExecutionResult = {
-    val tag = makeTag(backendCall)
-    val jesParameters = standardParameters(backendCall.callGcsPath) ++ gcsAuthParameter(backendCall.workflowDescriptor) ++ jesInputs ++ jesOutputs
-    logger.info(s"$tag `$command`")
-    JesConnection.storage.uploadObject(backendCall.gcsExecPath, command)
+  @annotation.tailrec
+  private def createJesRun(backendCall: BackendCall, jesParameters: Seq[JesParameter], retries: Int, delaySeconds: Int): Try[Run] = {
+    Try {
+      val pipeline = Pipeline(
```

Comment:
Yeah I believe I have a stack trace somewhere that shows that this line throws an exception:

```scala
val pipelineId = jesConnection.genomics.pipelines().create(cpr).execute().getPipelineId
```